Sources/SwiftKuery/Table.swift

```diff
@@ -101,32 +139,107 @@ open class Table: Buildable {
+        
+        if let primaryKey = primaryKey {
+            query += ", PRIMARY KEY ("
+            query += primaryKey.map { Utils.packName($0.name, queryBuilder: queryBuilder) }.joined(separator: ", ")
+            query += ")"
+        }
+        
+        if let foreignKeyColumns = foreignKeyColumns, let foreignKeyReferences = foreignKeyReferences {
+            query += ", FOREIGN KEY ("
+            query += foreignKeyColumns.map { Utils.packName($0.name, queryBuilder: queryBuilder) }.joined(separator: ", ")
+            query += ") REFERENCES "
+            let referencedTableName = foreignKeyReferences[0].table._name
+            query += Utils.packName(referencedTableName, queryBuilder: queryBuilder) + "("
+            query += foreignKeyReferences.map { Utils.packName($0.name, queryBuilder: queryBuilder) }.joined(separator: ", ")
+            query += ")"
+        }
+        
+        query += ")"
+        return query
+    }
+    
+    // MARK: Create Alias
+    /**
+     Function to return a copy of the current `Table` instance with the given name as its alias.
+     This is equivalent to the SQL AS operator.
+     ### Usage Example: ###
+     In this example, a `Table` instance is created. An alias for this `Table` instance is then created and its alias printed.
+     ```swift
+     let todotable = ToDoTable()
+     let aliasTable = todotable.as("new name")
+     print(String(describing: aliasTable.alias))
+     //Prints Optional("new name")
+     ```
+     */
     /// - Parameter newName: A String containing the alias for the table.
     /// - Returns: A new Table instance with the alias.
     public func `as`(_ newName: String) -> Self {
         let new = type(of: self).init()
         new.alias = newName
         return new
     }
-    
-    /// Apply TRUNCATE TABLE query on the table.
-    ///
+    // MARK: Query Database
+
+    /**
+     Function to return a `Raw` instance, which will execute a TRUNCATE query on the current `Table` instance.
+     ### Usage Example: ###
+     In this example, a `Table` instance is created. The truncate function is called to create a `Raw` instance of an String to execute the TRUNCATE SQL Query for todotable.
+     ```swift
+     let todotable = ToDoTable()
+     let truncateRaw = todotable.truncate()
+     print(truncateRaw))
+     //Prints Raw(query: "TRUNCATE TABLE", tables: [Application.ToDoTable])
+     ```
+     */
     /// - Returns: An instance of `Raw`.
     public func truncate() -> Raw {
         return Raw(query: "TRUNCATE TABLE", table: self)
     }
     
-    /// Apply DROP TABLE query on the table.
-    ///
+    /**
+     Function to return a `Raw` instance, which will execute a DROP TABLE query on the current `Table` instance.
+     ### Usage Example: ###
+     In this example, a `Table` instance is created. The drop function is called to create a `Raw` instance of an String to execute the DROP TABLE SQL Query for todotable.
```

Comment:
a String not an String

Reply:
This summary description is too complex and long. That could be in the notes underneath the API summary description if you like but we need something more like "Execute a DROP TABLE query on the current `Table` instance." or whatever this does.. it's very confusing having the text about a `Raw` instance as it then reads as though you'd have to do something else afterwards to execute the query.